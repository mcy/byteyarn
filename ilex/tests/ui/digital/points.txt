1/2/3e4/5
1/2/3/4e4/5
1/2e4/5
1/2/3e4/5/6
1/2/3e4