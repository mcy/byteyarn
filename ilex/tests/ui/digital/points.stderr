error: expected at least 2 `/`s
 --> digital/points.txt:2:7
  |
2 | 1/2/3/4e4/5
  |       ^
  |
  = note: reported at: ilex/src/rt/emit2.rs:554:16

error: unrecognized character
 --> digital/points.txt:2:6
  |
2 | 1/2/3/4e4/5
  |      ^
  |
  = note: reported at: ilex/src/rt/mod.rs:36:8

error: expected at least 2 `/`s
 --> digital/points.txt:3:1
  |
3 | 1/2e4/5
  | ^^^
  |
  = note: reported at: ilex/src/rt/emit2.rs:554:16

error: expected at least 2 `/`s
 --> digital/points.txt:4:11
  |
4 | 1/2/3e4/5/6
  |           ^
  |
  = note: reported at: ilex/src/rt/emit2.rs:554:16

error: unrecognized character
 --> digital/points.txt:4:10
  |
4 | 1/2/3e4/5/6
  |          ^
  |
  = note: reported at: ilex/src/rt/mod.rs:36:8

error: expected at least 1 `/`
 --> digital/points.txt:5:6
  |
5 | 1/2/3e4
  |      ^^
  |
  = note: reported at: ilex/src/rt/emit2.rs:554:16

error: aborting due to 6 errors
